.github/workflows/integration.yaml

```diff
@@ -167,3 +167,40 @@ jobs:
             unicode \
             unicode/utf16 \
             unicode/utf8
+
+
+  wasi-testsuite:
+    name: wasi-testsuite
+    runs-on: ubuntu-20.04
```

Comment:
let's run mac and windows as well!

Reply:
oh! 😬

Reply:
is that     `runs-on: ${{ matrix.os }}` ?

Reply:
yep

Reply:
missing Python module on macOS:
- ModuleNotFoundError: No module named 'colorama'
[18](https://github.com/tetratelabs/wazero/actions/runs/4004990710/jobs/6874809700#step:7:19)

path syntax error on windows?
- go install ./cmd/wazero
[3](https://github.com/tetratelabs/wazero/actions/runs/4004990710/jobs/6874809815#step:5:3)